src/HCSR04.h

```diff
@@ -13,25 +13,32 @@ class UltraSonicDistanceSensor {
     /**
      * @param triggerPin  Digital pin that is used for controlling sensor (output).
      * @param echoPin  Digital pin that is used to get information from sensor (input).
+     * @param maxDistanceCm  Maximum distance sensor can measure, defaults to 4m for HC-SR04.
+     *                       Set to maximum measuring distance to reduce measurement time.
+     * @param maxTimeoutMicroSec  Set absolute measurement timeout.
+                             Useful if you need measurement will not take longer than some specific time.
+                             This also means that we are limiting max distance it can measure.
```

Comment:
```suggestion
     * @param maxTimeoutMicroSec  Single measurement will never take longer than whatever value you provide here.
     *   You might want to do this in order to ensure your program is never blocked for longer than some specific time,
     *   since measurements are blocking.
     *   By default, there is no limit on time (only on distance). By defining timeout, you are again limiting the distance. 
```